### DISCUSSION TITLE
posts/unifi-express/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#284] [Comments]
# posts/unifi-express/

The UniFi Express from Ubiquiti is here and it’s going to shake up how we connect small and home networks. It’s a gateway that has WiFi 6 that runs the UniFi network application and can transform into an access point or mesh when your network grows. We’ll be taking a look at this device, its features, how to configure and manage it, and discuss all the uses cases for this versatile device.

https://technotim.live/posts/unifi-express/

**Comment by jtikkis** [comment]
Nice review. A lot of useful information. One question about UniFi Express as travel router: Can it be connected to wireless network (without ethernet connection) and provide secure, private wireless network for you devices ("travel office")? If yes, I assume that site-to-site VPN is also possible?

**Reply by selfjc** [reply]
I'm also wondering about this for setting up an RV Wi-Fi that connects to a campground's Wi-Fi for Internet/gateway routing but keeps another Wi-Fi network separate and private for Chromecast, laptops, phones, etc.
